lib/search-bar/index.jsx

```diff
@@ -25,7 +25,12 @@ export const SearchBar = ({
   showTrash,
 }) => (
   <div className="search-bar theme-color-border">
-    <IconButton icon={<MenuIcon />} onClick={onToggleNavigation} title="Menu" />
+    <IconButton
+      icon={<MenuIcon />}
+      disableTooltip={true}
```

Comment:
Why did you disable it for the menu button?

Reply:
I was under the impression that it was a standard Material design pattern to disable tooltips on the hamburger button, but I did a little more research today and it doesn't really seem to be the case 🤔 (There's even variance in Google's own apps!)

I'm going to remove this logic to keep it simple. Thanks!